tests/test_vrep_single_joint.py

```diff
@@ -1,15 +1,56 @@
 from robot_player import MotionManager, VrepOptions
+import numpy as np
+import csv
+from itertools import zip_longest
+import time
 
 motor_id = [1]
 dt = .01
+effort = []
+position = []
+times = []
 
-with MotionManager(motor_id, dt, options=VrepOptions()) as mm:
-    mm.initialize()
+with open('/home/alexander/Desktop/Acceleration_Calculation.csv', 'w', newline='') as A_Calculation:
```

Comment:
We should make a new file for this and keep this test. Also, you have an absolute path to a file on your computer which will not work on other computers. You should use a relative path to accomplish this instead if you want this functionality to be accessible across other computers.